npm_and_yarn/helpers/lib/yarn/subdependency-updater.js

```diff
@@ -39,6 +39,7 @@ async function getLockfileEntryForUpdate(depName, depVersion) {
     cwd: directory,
     nonInteractive: true,
     enableDefaultRc: true,
+    extraneousYarnrcFiles: [".yarnrc"],
```

Comment:
This is the only documentation I've found for this https://github.com/yarnpkg/yarn/blob/25d5526c8995443b03c113c44f9c8e9a79c2c074/src/config.js#L57